CMakeLists.txt

```diff
@@ -398,3 +398,11 @@ if(UNIX)
   install(TARGETS uv LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
   install(TARGETS uv_a ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
 endif()
+
+if(WIN32)
+  install(DIRECTORY include/ DESTINATION include)
+  install(FILES LICENSE DESTINATION .)
+  install(TARGETS uv uv_a
+    RUNTIME DESTINATION lib/$<CONFIG>
+    ARCHIVE DESTINATION lib/$<CONFIG>)
```

Comment:
Tiniest of style nits: can you line these up with the TARGETS keyword? Otherwise LGTM.

Reply:
Done.